### DISCUSSION TITLE
Welcome to AzureStorageAccount Discussions!

### DISCUSSION THREAD

**Discussion by EmreOzanMemis** [EmreOzanMemis/AzureStorageAccount#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1web1medya** [comment]
AWS S3 Strage dan Azure Storage' nasıl veri taşıyabilirim?

**Reply by EmreOzanMemis** [reply]
Azure Data Factory (ADF), veri taşımak için tercih edilen güçlü bir servis olarak karşımıza çıkıyor. 
Azure Portal'dan yeni bir Azure Data Factory kaynağı oluşturarak sürece başlayabilirsiniz. sonrasında Linked Service oluşturun erişim bilgilerinizi tanımlayın. data pipeline oluşturup hedef ve kaynağınızı belirterek kolayca yapabilrisiniz. 

Bir diğer yöntemde Azure Storage Migration Tool  yada AZCopy kullanılarak yapılabilir.

**Reply by 1web1medya** [reply]
harika
